PowerEditor/installer/nativeLang/dutch.xml

```diff
@@ -1386,6 +1404,7 @@ Wilt u het openen?"/>
             <CreateNewFileOrNot title="Nieuw bestand aanmaken" message="&quot;$STR_REPLACE$&quot; bestaat niet. Bestand aanmaken?"/>
             <CreateNewFileError title="Nieuw bestand aanmaken" message="Kan bestand &quot;$STR_REPLACE$&quot; niet aanmaken."/> <!-- HowToReproduce: this message prevents from system failure. It's hard to reproduce. -->
             <OpenFileError title="FOUT" message="Kan bestand &quot;$STR_REPLACE$&quot; niet openen."/>
+            <OpenFileNoFolderError title="Kan bestand niet openen" message="&quot;$STR_REPLACE1$&quot; kan niet worden geopend:
```

Comment:
PowerEditor/installer/nativeLang\dutch.xml: PowerEditor/installer/nativeLang/dutch.xml:1408:13:FATAL:PARSER:ERR_LT_IN_ATTRIBUTE: Unescaped '<' not allowed in attributes values PowerEditor/installer/nativeLang/dutch.xml:1408:13:FATAL:PARSER:ERR_SPACE_REQUIRED: attributes construct error PowerEditor/installer/nativeLang/dutch.xml:1408:13:FATAL:PARSER:ERR_GT_REQUIRED: Couldn't find end of Start Tag OpenFileNoFolderError line 1407: XMLSyntaxError Invalid File